api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -43,7 +43,9 @@
      * can, however, override this behaviour by providing absolute URLs
      * ourselves.
      */
-    private static final String CONVERSATIONS_BASE_URL = "https://conversations.messagebird.com/v1";
+    private String CONVERSATIONS_BASE_URL  = "https://conversations.messagebird.com/v1";
```

Comment:
can you remove the extra space before =?

Reply:
On it